manifests/Microsoft/VisualStudioCodeInsiders/1.46.0.yaml

```diff
@@ -0,0 +1,13 @@
+Id: Microsoft.VisualStudioCodeInsiders
+Version: 1.46.0
+Name: Visual Studio Code Insiders
+Publisher: Microsoft Corporation
+License: Copyright (c) Microsoft Corporation
+AppMoniker: vscode-insiders
```

Comment:
I want to get https://github.com/microsoft/winget-pkgs/issues/277 sorted out before we do insiders, ideally we would use `code` and `code-insiders` to match the CLI and have `vscode` as an alias/search term.

Reply:
Thanks, I added [my opinion](https://github.com/microsoft/winget-pkgs/issues/277#issuecomment-631440881) on it!